### PR TITLE
Homepage, Navbar and Refactor

### PR DESCRIPTION
**Pull request by varun13169**

### CODE REVIEW

File: src/components/Carousal/Carousal.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import "./carousal.css";
+import { useState } from "react";
+
+import carousal_1 from "./carousal_1.jpg";
+// import carousal_2 from "./src/assets/imgaes/carousal_2";
+// import carousal_3 from "./src/assets/imgaes/carousal_3";
```
Comment (zoef-yt) [inline src/components/Carousal/Carousal.jsx:6]: Remove commented code.

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -1,110 +1,155 @@
...
+            stroke-width="1.5"
+            stroke="#059669"
+            fill="none"
+            stroke-linecap="round"
+            stroke-linejoin="round"
+          >
+            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+            <circle cx="10" cy="10" r="7" />
+            <line x1="21" y1="21" x2="15" y2="15" />
+          </svg>
```
Comment (zoef-yt) [inline src/components/Navbar/Navbar.jsx:31]: You can import SVG's in one file and import SVG from there.

**Review by zoef-yt — APPROVED** [review]
The code looks good. Do add description it's empty. Good to merge.

**Review by parulgupta2001 — APPROVED** [review]
Remove commented code, otherwise everything looks good.

**Pull request merged**